Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -151,13 +151,26 @@
     <string name="risk_card_low_risk_no_encounters_body">Keine Risiko-Begegnungen</string>
     <!-- XTXT: risk card - Low risk state - Days with low risk encounters -->
     <plurals name="risk_card_low_risk_encounter_days_body">
+        <item quantity="one">"Begegnungen mit niedrigem Risiko an %1$d Tag"</item>
+        <item quantity="other">"Begegnungen mit niedrigem Risiko an %1$d Tagen"</item>
+        <item quantity="zero">"Begegnungen mit niedrigem Risiko an %1$d Tagen"</item>
```

Comment:
```suggestion
        <item quantity="zero">"Keine Risiko-Begegnungen"</item>
```
Should be: "Keine Risiko-Begegnungen" because it makes no sense to write "Begegnungen mit niedrigem Risiko an 0 Tagen"